setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsfileLinterLauncher.java

```diff
@@ -0,0 +1,78 @@
+package io.jenkins.jenkinsfile.runner;
+
+import hudson.security.ACL;
+import io.jenkins.cli.shaded.org.apache.commons.io.FileUtils;
+import io.jenkins.jenkinsfile.runner.bootstrap.commands.LintJenkinsfileCommand;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.charset.StandardCharsets;
+import org.codehaus.groovy.control.ErrorCollector;
+import org.codehaus.groovy.control.MultipleCompilationErrorsException;
+import org.codehaus.groovy.control.messages.SyntaxErrorMessage;
+
+/**
+ * Set up of Jenkins environment for linting a single Jenkinsfile.
+ */
+public class JenkinsfileLinterLauncher extends JenkinsLauncher<LintJenkinsfileCommand> {
+
+    public static final String CONVERTER_CLASS_NAME = "org.jenkinsci.plugins.pipeline.modeldefinition.parser.Converter";
+
+    public JenkinsfileLinterLauncher(LintJenkinsfileCommand command) {
+        super(command);
+    }
+
+    //TODO: add support of timeout
+
+    /**
+     * Launch the Jenkins instance
+     * No time out and no output message
+     */
+    @Override
+    protected int doLaunch() throws Exception {
+        // So that the payload code has all the access to the system
+        ACL.impersonate(ACL.SYSTEM);
```

Comment:
This is the deprecated call, right? Better to do try-with-resources method so that we do not leak the context

Reply:
Can't do a try-with-resources with Java 8, so swapped to a try-finally block.

Reply:
Wait! What?  Yes, you can do a try-with-resource at Java 8 🤦 
My IDE is lying.  One second, let me check that again.

Reply:
Ok. Changed it to a try-with-resources.  Also took the liberty of changing `JenkinsfileRunnerLauncher` too, so that both files are consistent.